osf/utils/machines.py

```diff
@@ -287,7 +287,16 @@ def notify_submit(self, ev):
             reviews_signals.reviews_email_withdrawal_requests.send(timestamp=timezone.now(), context=context)
```

Comment:
Is the only type of PreprintRequest a *withdrawal* request?  Is this machine going to be expanded in the future to support other types of requests?   

On one hand, this looks like it was written to originally be general so it could work for many things, but then we have very withdrawal-specific behavior here.

Reply:
@pattisdr My original intention was that PreprintRequest is always withdrawal request. I didn't consider expanding it for future support of other types of requests.
Originally, there was only `RequestMachine` and `RequestAction`, but the behavior was purely for Node access request. So I changed it to `NodeRequestMachine` and `NodeRequestAction`, and added `PreprintRequestMachine` and `PreprintRequestAction`. In hindsight, the names could be more specific, something like `NodeAccessRequestAction` and `PreprintWithdrawalRequestAction`.

Reply:
okay! thank you for clearing that up - could you just add a quick comment to `PreprintRequest` that this is for withdrawals only?

Reply:
@pattisdr Comments are added to NodeRequest and PreprintRequest.